references/detection/evaluate_pytorch.py

```diff
@@ -130,7 +131,11 @@ def main(args):
         model = model.cuda()
 
     # Metrics
-    metric = LocalizationConfusion(use_polygons=args.rotation, mask_shape=input_shape)
+    metric = LocalizationConfusion(
+        use_polygons=args.rotation,
+        mask_shape=input_shape,
+        use_broadcasting=True if int(psutil.virtual_memory().total / 1024**3) > 63 else False,
```

Comment:
I think that's a dangerous "non-human" conditional arg

Reply:
yeah i agree :sweat_smile: will make it more readable